--- polkit-kde-agent-1-5.4.1/AuthDialog.cpp.omv~	2015-10-02 20:17:37.691152284 +0200
+++ polkit-kde-agent-1-5.4.1/AuthDialog.cpp	2015-10-02 20:23:13.300440452 +0200
@@ -67,9 +67,9 @@ AuthDialog::AuthDialog(const QString &ac
 
     setupUi(this);
 
-    connect(buttonBox, &QDialogButtonBox::accepted, this, &AuthDialog::okClicked);
-    connect(buttonBox, &QDialogButtonBox::accepted, this, &QDialog::accept);
-    connect(buttonBox, &QDialogButtonBox::rejected, this, &QDialog::reject);
+    connect(buttonBox, SIGNAL(accepted()), this, SIGNAL(okClicked()));
+    connect(buttonBox, SIGNAL(accepted()), this, SLOT(accept()));
+    connect(buttonBox, SIGNAL(rejected()), this, SLOT(reject()));
 
     QString detailsButtonText = i18n("Details");
     QPushButton* detailsButton = new QPushButton(detailsButtonText + " >>");
--- polkit-kde-agent-1-5.4.1/main.cpp.omv~	2015-10-02 20:26:46.752538469 +0200
+++ polkit-kde-agent-1-5.4.1/main.cpp	2015-10-02 20:29:31.050416098 +0200
@@ -49,12 +49,8 @@ int main(int argc, char *argv[])
     PolicyKitKDE agent(argc, argv);
     KDBusService service(KDBusService::Unique);
 
-    auto disableSessionManagement = [](QSessionManager &sm) {
-        sm.setRestartHint(QSessionManager::RestartNever);
-    };
-
-    QObject::connect(&agent, &QGuiApplication::commitDataRequest, disableSessionManagement);
-    QObject::connect(&agent, &QGuiApplication::saveStateRequest, disableSessionManagement);
+    QObject::connect(&agent, SIGNAL(commitDataRequest(QSessionManager&)), &agent, SLOT(disableSessionManagement(QSessionManager&)));
+    QObject::connect(&agent, SIGNAL(saveStateRequest(QSessionManager&)), &agent, SLOT(disableSessionManagement(QSessionManager&)));
 
     agent.exec();
 }
--- polkit-kde-agent-1-5.4.1/policykitkde.cpp.omv~	2015-10-02 20:30:22.482480354 +0200
+++ polkit-kde-agent-1-5.4.1/policykitkde.cpp	2015-10-02 20:30:44.251077334 +0200
@@ -47,3 +47,8 @@ PolicyKitKDE::~PolicyKitKDE()
 {
     m_listener->deleteLater();
 }
+
+
+void PolicyKitKDE::disableSessionManagement(QSessionManager &sm) {
+    sm.setRestartHint(QSessionManager::RestartNever);
+};
--- polkit-kde-agent-1-5.4.1/policykitkde.h.omv~	2015-10-02 20:29:51.220142956 +0200
+++ polkit-kde-agent-1-5.4.1/policykitkde.h	2015-10-02 20:30:20.918886688 +0200
@@ -22,6 +22,7 @@
 #define POLICYKITKDE_H
 
 #include <QApplication>
+#include <QSessionManager>
 
 #include "policykitlistener.h"
 
@@ -31,6 +32,8 @@ class PolicyKitKDE : public QApplication
 public:
     PolicyKitKDE(int &argc, char **argv);
     virtual ~PolicyKitKDE();
+public slots:
+    void disableSessionManagement(QSessionManager &sm);
 private:
     PolicyKitListener *m_listener;
 };
